### PR TITLE
Make tour battles inherit mutes

### PR DESCRIPTION
**Pull request by whalemer**

### CODE REVIEW

File: server/rooms.ts [hunk 1]
```diff
@@ -1447,6 +1447,12 @@ export class GameRoom extends BasicChatRoom {
 
 		this.tour = options.tour || null;
 		this.parent = options.parent || (this.tour && this.tour.room) || null;
+		// copy mutes from parent room
```
Comment (Zarel) [inline server/rooms.ts]: I would rather inherit them (search parents for mutes when checking for mutes) than copy them. That way, new mutes would also apply to subrooms.

Comment (whalemer) [reply server/rooms.ts]: .....yeah, that's a much better way to do this

File: server/rooms.ts [hunk 2]
```diff
@@ -373,7 +375,7 @@ export abstract class BasicRoom {
 		}
 		this.runMuteTimer();
 
-		user.updateIdentity(this.roomid);
+		user.updateIdentity();
```
Comment (Zarel) [inline server/rooms.ts]: ?

Comment (whalemer) [reply server/rooms.ts]: You need to update it globally now, since you might also be getting muted / unmuted in a subroom.

Comment (Zarel) [reply server/rooms.ts]: ohhhh

**Pull request merged**